packages/xo-web/package.json

```diff
@@ -60,6 +60,7 @@
     "complex-matcher": "^0.5.0",
     "cookies-js": "^1.2.2",
     "copy-to-clipboard": "^3.0.8",
+    "crypto": "^1.0.1",
```

Comment:
Not necessary, the builder (Browserify) will provide a browser implementation automatically.